widget/embedded/src/containers/WidgetInfo/WidgetInfo.types.ts

```diff
@@ -13,7 +13,7 @@ export interface WidgetInfoContextInterface {
     details: ConnectedWallet[];
     totalBalance: string;
     isLoading: boolean;
-    refetch: (accounts: Wallet[], tokens: Token[]) => void;
```

Comment:
this can be a breaking change since the hook is public. i don't know why we'd got a second param and didn't use in implementation.

Reply:
It's needed in dApp. We shouldn't break it.

Reply:
Thanks for the heads up. I opened a PR there.